### ISSUE TITLE
Replace preserve-merges with rebase-merges

### ISSUE THREAD

**Issue by alisonjo315** [SethRobertson/GitFixUm#28]
`--preserve-merges` (aka `-p`) isn't supported, as of Git 2.35.0:
https://developercommunity.visualstudio.com/t/Git-no-longer-supports-preserve-merges/1671071

Today, I was [removing an entire commit](https://sethrobertson.github.io/GitFixUm/fixup.html#remove_deep) via `git rebase -p --onto SHA^ SHA`, and ran into this error:
```
fatal: --preserve-merges was replaced by --rebase-merges
```

Fortunately, all I had to do was replace `-p` with `-r`!

I know this repo hasn't had an update in a while, but I figure, maybe this will help someone -- and, what the heck, I'll submit a PR just in case.

----

TL;DR: To remove an entire commit, the command is:
```
git rebase -r --onto SHA^ SHA
```
(yes: `-r` &nbsp; &ndash; &nbsp; no: `-p`)

**Comment by alisonjo315** [comment]
Thank you!
